Primer PR

camreynolds opened the pull request:
Correcto!

neoluciano committed:
Primer PR

neoluciano committed:
ok
Merge branch 'master' of https://github.com/alejandroarevalov/holamundo into lpereyra

alejandroarevalov reviewed (changes requested):
Por favor modificar indentación. El tag H1 y H2 deben estar a la misma altura

neoluciano committed:
commit

neoluciano committed:
Merge branch 'master' of https://github.com/alejandroarevalov/holamundo into lpereyra

neoluciano committed:
ok
Merge branch 'master' of https://github.com/alejandroarevalov/holamundo into lpereyra

alejandroarevalov reviewed (approved):
Approved
